src/browser/provider/built-in/dedicated/chrome/cdp-client/index.ts

```diff
@@ -232,6 +232,50 @@ export class BrowserClient {
         }
     }
 
+    public async maximizeWindow (): Promise<void> {
+        const target = await getFirstTab(this._port);
+        const client = await this.getActiveClient();
+
+        if (client) {
+            const windowParams = await client.Browser.getWindowForTarget({ targetId: target.id });
+
+            await client.Browser.setWindowBounds({ windowId: windowParams.windowId, bounds: { windowState: 'maximized' } });
+        }
+    }
+
+    public async resizeBounds (newDimensions: Size): Promise<void> {
+        const { viewportSize, config } = this._runtimeInfo;
+
+        const horizontal = viewportSize.outerWidth - viewportSize.width;
+        const vertical = viewportSize.outerHeight - viewportSize.height;
+
+        const target = await getFirstTab(this._port);
+        const client = await this.getActiveClient();
+
+        if (client) {
+            await this._setDeviceMetricsOverride(client, newDimensions.width, newDimensions.height, 1, config.mobile);
+
+            const windowParams = await client.Browser.getWindowForTarget({ targetId: target.id });
+
+            if (windowParams.bounds.windowState !== 'normal') {
+                await client.Browser.setWindowBounds({
+                    windowId: windowParams.windowId,
+                    bounds:   {
+                        windowState: 'normal',
+                    },
+                });
+            }
```

Comment:
It's worth checking again whether setting this state is necessary for us. If it doesn't affect the operation at all, then remove it.